poly/src/evaluations/multivariate/multilinear/dense.rs

```diff
@@ -247,7 +247,7 @@ impl<F: Field> Add for DenseMultilinearExtension<F> {
     }
 }
 
-impl<'a, 'b, F: Field> Add<&'a DenseMultilinearExtension<F>> for &'b DenseMultilinearExtension<F> {
+impl<'a, F: Field> Add<&'a DenseMultilinearExtension<F>> for &DenseMultilinearExtension<F> {
```

Comment:
Which lifetime does the `&DenseMultilinearExtension` get here? `'a` or an anonymous one?

Reply:
It's an anonymous lifetime directly inferred by the compiler :)